addons/sale_loyalty/tests/test_program_with_code_operations.py

```diff
@@ -411,3 +411,54 @@ def test_edit_and_reapply_promotion_program(self):
         self._apply_promo_code(order, 'test')
         # But the above line should not add any reward
         self.assertEqual(len(order.order_line), 2, "You should get a discount line") # product + discount
+
+    def test_multiple_rewards_per_program(self):
+
+        coupon_program_one = self.env['loyalty.program'].create({
+            'name': 'First coupon',
+            'program_type': 'coupons',
+            'reward_ids': [(0, 0, {
+                'reward_type': 'discount',
+                'discount_mode': 'percent',
+                'discount': 10,
+                })],
+            })
```

Comment:
```suggestion
```
Why you are not used existing record `self.code_promotion_program_with_discount` instead?